app/imports/ui/components/NavBar.jsx

```diff
@@ -57,11 +69,17 @@ class NavBar extends React.Component {
 /** Declare the types of all properties. */
 NavBar.propTypes = {
   currentUser: PropTypes.string,
+  profile: PropTypes.array,
+  ready: PropTypes.bool.isRequired,
+
 };
 
 /** withTracker connects Meteor data to React components. https://guide.meteor.com/react.html#using-withTracker */
+const subscription = Meteor.subscribe('Profiles');
 const NavBarContainer = withTracker(() => ({
   currentUser: Meteor.user() ? Meteor.user().username : '',
+  profile: Profiles.find({ owner: Meteor.user() ? Meteor.user().username : '' }).fetch(),
```

Comment:
pull Meteor.user() out into a const first then use the value in the find method. 

```
const username = Meteor.user().username;
...
...

profile: Profiles.find({ owner: username }).fetch()
```

Reply:
Unfortunately that won't work.  First off the value needs to be 
```Meteor.user() ? Meteor.user().username : ''``` 
but I'm honestly not sure why.  And even if I assigned the correct value to 'const username':
```
const username = Meteor.user() ? Meteor.user().username : '';
...
profile: Profiles.find({ owner: username }).fetch()
```
this won't work.  It says this.props.profile is undefined but I'm not sure why.  I'll ask Branden about this tomorrow.  For now though I'm just going to leave it as is since it works for all roles.